CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
-## ChangeLog
+# Changelog
+All notable changes to this project will be documented in this file.
 
-## 0.1.0
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+## [0.2.0] - 2020-10-28
```

Comment:
Should we release this as 0.13.0 to track upstream?

@jodeev thoughts?

Reply:
Yep, sounds like we're going to be tracking upstream as well as possible, while making an honest attempt to preserve semver at the same time.